Makefile

```
@@ -1,6 +1,9 @@
 NAME          := hdf5
-SRC_EXT       := bz2
+SRC_EXT       := gz
 TEST_PACKAGES := $(NAME) java-$(NAME) $(NAME)-devel $(NAME)-static $(NAME)-mpich-tests $(NAME)-openmpi3-tests
+DL_VERSION    := fa40c6c59af5d9aabd4b478cd02f8a9f7ebf7922
+GIT_SHORT     := $(shell git rev-parse --short $(DL_VERSION))
+BUILD_DEFINES := --define "%relval .g$(GIT_SHORT)" --define "%hdf5_commit $(DL_VERSION)"
```

When you define macros, you don't include the `%` in the name:
```suggestion
BUILD_DEFINES := --define "relval .g$(GIT_SHORT)" --define "hdf5_commit $(DL_VERSION)"
```